### DISCUSSION TITLE
Obtain Source from DNS queries

### DISCUSSION THREAD

**Discussion by nyevess** [PowerDNS/pdns#13216] [Q&A]
Hello, 
 I have 2 powerDNS server with DB and 3 recursor server, I need to log all DNS queries to obtain the IP source of this queries, do you know how I can obtain this information?
Thanks!!

**Comment by omoerbeek** [comment]
As you asked on IRC the same time, could you summarize the answers? That way, it becomes findable for others.

**Comment by klaus-nicat** [comment]
We stopped using logging features of the servers and instead are using packetbeat (elastic) to track all queries and responses on our resolvers. That might help as well.
